### ISSUE TITLE
Add `difficulty` property to `HowTo` schema

### ISSUE THREAD

**Issue by astappiev** [schemaorg/schemaorg#3130]
There is already [educationalLevel](https://schema.org/educationalLevel) inherited from [CreativeWork](https://schema.org/CreativeWork), but the reference to education is confusing, usually it's more about skill level and not education.

The property should support all types ([DefinedTerm](https://schema.org/DefinedTerm)  or [Text](https://schema.org/Text)  or [URL](https://schema.org/URL)) of [educationalLevel](https://schema.org/educationalLevel).

An example values would be:
- easy
- moderate
- difficult

Or maybe:
- beginner
- advanced
- chef

**Comment by philbarker** [comment]
@astappiev While I agree that the reference to education seems out of place, your examples are all good examples of values for `educationalLevel`. `competenceRquired` refers to the actual ability (not the level) that that is necessary, e.g. must be able to "chop herbs", "beat eggs", "fry in a skillet". If you think educationalLevel is completely unsuitable I guess you'll need a new property like `difficulty` for HowTo

**Comment by astappiev** [comment]
Yes, I also agree that the examples works good for `educationalLevel`.
But it's odd to have `educationalLevel` in the `Recipe` schema, because it's more about skill and not education.

The approach with `difficulty` would be better, I guess, but I followed the simplest way.
